Web Integration/Tests/FiftyOne.Pipeline.Web.Shared.Tests/FiftyOne.Pipeline.Web.Shared.Tests.csproj

```diff
@@ -4,14 +4,18 @@
     <TargetFramework>net6.0</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <StartupObject></StartupObject>
+
+    <IsPublishable>True</IsPublishable>
```

Comment:
Is this addition necessary? I haven't looked into how this is used by dotnet, but it could result in the test package being released to NuGet.

Reply:
Removed these lines in commit just now.